boards/imix/src/aes_test.rs

```diff
@@ -0,0 +1,24 @@
+use capsules::test::aes::Test;
+use kernel::hil::symmetric_encryption::{AES128, AES128_BLOCK_SIZE};
+use sam4l::aes::{AES, Aes};
+
+pub fn run() {
+    let t = static_init_test();
+
+    unsafe {
+        AES.set_client(t);
+    }
+
+    t.run();
+}
+
+fn static_init_test() -> &'static mut Test<'static, Aes<'static>> {
+    unsafe {
+        let source = static_init!([u8; 4 * AES128_BLOCK_SIZE], [0; 4 * AES128_BLOCK_SIZE]);
+        let data = static_init!([u8; 6 * AES128_BLOCK_SIZE], [0; 6 * AES128_BLOCK_SIZE]);
```

Comment:
Why is the `data` buffer 6 blocks instead of 4 blocks?

Reply:
Again, I think these are some arbitrary sizes to facilitate testing the ability to encrypt a range into a different part of a slice, but I might be wrong.